agent/eni/iphelperwrapper/iphelper_windows.go

```diff
@@ -0,0 +1,155 @@
+// +build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package iphelperwrapper
+
+import (
+	"C"
+	"math"
+	"syscall"
+	"time"
+	"unsafe"
+
+	log "github.com/cihub/seelog"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows"
+)
+
+const (
+	// After registering a callback with NotifyIPInterfaceChange API, we get an initial notification to confirm that the callback is registered
+	// All other notifications sent on the channel would be positive except this one
+	InitialNotification = math.MinInt16
+	// This is the timeout for the initial notification. If notification is not received within this timeout then we will assume error
+	initialNotificationTimeout = 4 * time.Second
```

Comment:
can you make this a var and override it to small value in test? currently unit tests in iphelperwrapper take 15s because of this which i think is a bit too long